config/database.yml

```diff
@@ -10,7 +10,7 @@ common: &common
 
 development:
   <<: *common
-  database: <%= ENV.fetch("POSTGRES_DB", Settings.postgres.database ) %>_development
+  database: <%= ENV.fetch("POSTGRES_DB", Settings.postgres.database ) %>
```

Comment:
Can this be changed back to have the "_development" on the end?  That's the standard for rails.  This change will probably break our development environments.